### PR TITLE
fix(val-743): Operator name is not shown

### PR DESCRIPTION
**Pull request by AlexandrMov**
See details here: https://linear.app/lidofi/issue/VAL-743/operator-name-is-not-shown

### CODE REVIEW

File: ethereum-validators-set/src/subagents/node-operators-registry/agent-node-operators-registry.ts [hunk 1]
```diff
@@ -140,6 +140,12 @@ class NodeOperatorsRegistryModuleContext {
     await this.updateNodeOperatorsNames(currentBlock);
   }
 
+  getOperatorName(nodeOperatorId: string): string {
+    return (
+      this.nodeOperatorNames.get(Number(nodeOperatorId))?.name ?? "undefined"
```
Comment (avsetsin) [inline ethereum-validators-set/src/subagents/node-operators-registry/agent-node-operators-registry.ts:145]:
```suggestion
      this.nodeOperatorNames.get(Number(nodeOperatorId))?.name ?? `Operator #${nodeOperatorId}`
```

File: ethereum-validators-set/e2e/utils.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export function provideRunTransaction(
           `Error retrieving block number of transaction ${txHash}`,
         );
       }
-      await agent.initialize(tx.blockNumber);
+      await agent.initialize(initBlock ? initBlock : tx?.blockNumber);
```
Comment (avsetsin) [inline ethereum-validators-set/e2e/utils.ts:46]:
```suggestion
      await agent.initialize(initBlock == null ? tx?.blockNumber : initBlock);
```

Comment (avsetsin) [reply ethereum-validators-set/e2e/utils.ts:46]: `initBlock` is number and 0 is a valid block number

Comment (AlexandrMov) [reply ethereum-validators-set/e2e/utils.ts]: rewritten to check undefined, because initBlock is a number or undefined

**Pull request merged**